modules/images-configuration-shortname.adoc

```diff
@@ -90,7 +90,9 @@ status:
 When the `allowedRegistries` parameter is defined, all registries, including the `registry.redhat.io` and `quay.io` registries and the default {product-registry}, are blocked unless explicitly listed. If you use this parameter, to prevent pod failure, add all registries including the `registry.redhat.io` and `quay.io` registries and the `internalRegistryHostname` to the `allowedRegistries` list, as they are required by payload images within your environment. For disconnected clusters, mirror registries should also be added.
 ====
 
+ifndef::openshift-rosa,openshift-dedicated[]
```

Comment:
Nit/follow up(?):This `ifndef` makes this a one step procedure for ROSA and OSD. Technically, [line 40 above](https://github.com/openshift/openshift-docs/pull/63895/files#diff-6fb65b5ed05a3f67d1cee2098cc948d07c5d8d31c6aca5b234b350bd21dc48caL40) should be conditionalized to be a `*` for OSD and ROSA. Not sure if this is worth tackling in this PR or creating a minor follow up ticket.

Reply:
I also have no idea of how to do that or if it is even possible. I don't think it is a big deal if it is too hard to fix.

Reply:
@michaelryanpeter 

```
ifndef::openshift-rosa,openshift-dedicated[]
. Edit the `image.config.openshift.io/cluster` custom resource:
endif::openshift-rosa,openshift-dedicated[]
ifdef::openshift-rosa,openshift-dedicated[]
* Edit the `image.config.openshift.io/cluster` custom resource:
endif::openshift-rosa,openshift-dedicated[]
```

Reply:
Thank you for sharing the solution! I am glad it worked!